templates/zerver/help/link-to-a-message-or-conversation.md

```diff
@@ -77,6 +77,14 @@ message](/help/quote-and-reply).
 
 1. Click **Copy link to topic**.
 
+{tab|mobile}
```

Comment:
I think we should order Desktop > Web > Mobile (as a general pattern). It should work better, as on many pages Desktop/Web will have a shared set of instructions.

Reply:
Yes, that makes sense.